pkg/component/component.go

```diff
@@ -842,13 +844,37 @@ func GetComponentDesc(client *occlient.Client, componentName string, application
 	if err != nil {
 		return componentDesc, errors.Wrap(err, "unable to get envVars list")
 	}
+
+	linkedServices := make([]string, 0, 5)
+	linkedComponents := make(map[string][]string)
+	linkedSecretNames, err := GetComponentLinkedSecretNames(client, componentName, applicationName)
+	if err != nil {
+		return componentDesc, errors.Wrap(err, "unable to list linked secrets")
+	}
+	for _, secretName := range linkedSecretNames {
```

Comment:
This can be extracted into a function `filterSecrets` or something, if you think that could help.

Reply:
I don't think that would improve things for the time being, since it wouldn't be used anywhere else

Reply:
@girishramnani Can you please re-review in light of the latest comment? Thanks